prisma/schema.prisma

```diff
@@ -21,6 +21,20 @@ model Admin {
   @@map("admin")
 }
 
+model Member {
+  id String @id @default(cuid()) @db.VarChar(32)
+
+  name String
+
+  created_at DateTime @default(now())
+  updated_at DateTime @updatedAt
+
+  family   Family @relation(fields: [familyId], references: [id])
+  familyId String @db.VarChar(32)
```

Comment:
the convention for sql databases is to use snake_case for naming fields. so it's better to name this as `family_id`